ntc_templates/templates/cisco_ios_show_platform.textfsm

```diff
@@ -0,0 +1,12 @@
+Value INDEX (\d+)
+Value PORTS (\d+)
+Value MODEL (\S+)
+Value SERIAL (\S+)
+Value MAC (\S+)
+Value HW_VER (\S+)
+Value SW_VER (\d+\.\d+\.\d+)
+
+Start
+  ^Switch.*
+  ^--.*
```

Comment:
```suggestion
  ^-+\s*
```

Reduce dashes and match white space to make the regex slightly more specific.
Intentionally set for zero or more white space `\s*` just in case the hyphens ever end up being on a line without spaces.

Reply:
@Uncle-Dee
If you approve these suggestions, please click the button(s) to accept the Commit Suggestions.

Any questions?
Thank you.